extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -108,6 +110,7 @@ public function testWikiaMobileChangePasswordTemplate() {
 
 	public function loginDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "loginDataProvider" has 168 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)